### ISSUE TITLE
Feature Request: Non-admin creators

### ISSUE THREAD

**Issue by Deozaan** [itchio/itch.io#1271]
I sometimes collaborate with others on group projects, such as for game jams. I'm also usually the person who manages the game uploads and listing on itch.io. I'd like to be able to have the game we all work on appear on each of our itch profile pages. The only way to do that currently is to make everybody an admin of the game page. But that allows them to change anything about the game page, even add or remove other admins.

Ideally, I would like some kind of tiered permissions system so I could grant varying levels of access to modifying the game page and related details. But at a minimum, I just want to properly credit everyone who worked on the game by allowing them to choose whether or not it appears on their profile page, without giving them admin rights to change anything/everything else.

**Comment by Enemby** [comment]
Hi!

The 'proper' way to do this is by using itchio's Collections system. You're able to embed collections on your profile page by editing the theme. Using this you can highlight games you've worked on etc without having owner privileges. 

As an example: 
https://fracturedminddev.itch.io/

I too would like a tiered permission system, but hey! At least you can still do it.

**Comment by PyrateAkananto** [comment]
For a website like itch.io which supports collaboration in so many ways I am a bit baffled that you can list the accounts of contributors only by giving them full admin access to the game's page which is a step too drastic in my opinion. I second this feature request.

**Comment by hsandt** [comment]
... and backlinking to this itch community thread which details possible options for access granularity.

I got a real use case today: a few people contributed a little to the game without being part of core team. They will never have to edit page data, but I want them linked to the project.

Currently, the only way is to add them manually in some Special Thanks section in the page description (and ask them to add the game manually in their "Contributed to" collection as suggested above).

**Comment by secondepoch8** [comment]
I agree, this is a real problem - I had a case recently of a collaborator behaving badly, making unwanted changes to the game's page during the judgement phase of a game jam.  There needs to be a way of removing admin access without un-associating the person's name from the game (since they deserve credit for the aspect of the work that they did.)

Just putting them in the page description is a weak solution; having joint ownership of a game looks good for one's portfolio, having their name prominently displayed alongside that of the game and then taking that away later is hostile and problematic.

We need non-admin collaborators.
To date there has been no official comment on why this isn't possible (or philosophically undesired)

**Comment by joyce-stick** [comment]
this is also a feature I want. I prefer to be able to add collaborators as admins, but I want them to have limited control so that if, say, one of them has an episode of some kind and decides to delete all their work from the internet, they can't also just go and delete the game on my page that they contributed to without at the very least me having to approve that with the main owning account. Even if that never happens, what if their account is hacked? I don't want to have to even think of having to deal with the fallout of such a situation that could potentially reach far beyond their own page.
